pytorch_lightning/trainer/trainer.py

```diff
@@ -906,7 +909,9 @@ def call_hook(self, hook_name, *args, **kwargs):
     @staticmethod
     def available_plugins():
         """
-            List of all available plugins that can be string arguments to the trainer.
-            Returns: List of all available plugins that are supported as string arguments.
+        List of all available plugins that can be string arguments to the trainer.
+
+        Returns:
+            List of all available plugins that are supported as string arguments.
```

Comment:
Probably better to avoid this duplication

Reply:
Sorry, a bit confused. What is the change required here? I just changed the indentation by using the formatter.

Reply:
So the current docstring has

```
'''
Some text

Returns:
    The same text
'''
```

I was suggesting to remove the `returns:`